src/tennisSimulator/Menu.java

```diff
@@ -105,4 +108,53 @@ private String promptUserForGender() {
         System.out.print("Enter a gender (man/woman): ");
         return scanner.nextLine();
     }
+
+    private void selectPlayersForMatchSimulation() {
```

Comment:
This method is very long, for now I think let's leave it as is (because I think it might change with future code improvements), but it will definitely need refactoring in near future.

Reply:
Improved slightly for now.